test/factories/timer_session.rb

```diff
@@ -10,19 +10,22 @@
 
     trait :with_issues do
       after(:create) do |timer_session|
-        TimerSessionIssue.create!(
-          timer_session: timer_session,
-          issue: Issue.first
-        )
+        TimerSessionIssue.create!(timer_session:, issue: Issue.first)
       end
     end
 
     trait :with_time_entries do
       after(:create) do |timer_session|
-        TimerSessionTimeEntry.create!(
-          timer_session: timer_session,
-          time_entry: TimeEntry.first
-        )
+        TimerSessionTimeEntry.create!(timer_session:, time_entry: TimeEntry.first)
+      end
+    end
+
+    trait :with_unrounded_time_entries do
+      after(:create) do |timer_session|
+        3.times do
+          e = TimeEntry.create(user: User.current, project: Project.last, hours: 1.0 / 3, spent_on: Time.zone.today)
+          TimerSessionTimeEntry.create!(timer_session:, time_entry: e)
+        end
       end
     end
```

Comment:
Please move this to the test. You use it only once. Also I don't think the dependency on the user is wise to have in the factory. It should be agnostic.